addon-resizer/nanny/estimator.go

```diff
@@ -18,6 +18,7 @@ package nanny
 
 import (
 	"fmt"
+	"log"
```

Comment:
nanny_lib.go uses glog, can you use it here too, to stay consistent?

Reply:
Done.